.env.example

```
@@ -6,5 +6,7 @@ VECTORX_CONTRACT_ADDRESS=0xbc281367e1F2dB1c3e92255AA2F040B1c642ec75
 BRIDGE_CONTRACT_ADDRESS=0x1369a4c9391cf90d393b40faead521b0f7019dc5
 ETHEREUM_CLIENT_URL=https://ethereum-sepolia.publicnode.com
 BEACONCHAIN_URL=https://sepolia.beaconcha.in/api/v1/slot
+ETH_HEAD_RESPONSE_CACHE=240
```

```suggestion
ETH_HEAD_CACHE_MAXAGE=240
AVL_HEAD_CACHE_MAXAGE=600
```
Better to be a bit more verbose so it's clear what this variable is doing.